Cubical/Categories/Site/Sieve.agda

```diff
@@ -0,0 +1,105 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Site.Sieve where
+
+-- Definition of sieves on an object
+-- and the sieve generated by a cover.
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Structure
+
+open import Cubical.Functions.Logic using (∀[]-syntax; ∀[∶]-syntax; _⇒_; _⇔_)
```

Comment:
There is some hPropy stuff in this file as well. Maybe do the same as in `Site.Sheaf` for consistency's sake ;)

Reply:
Yes, indeed, I just didn't finish that file yet! :smile:

Reply:
Done.